### ISSUE TITLE
Inverter output constantly moves between 0 and 800W

### ISSUE THREAD

**Issue by Burny08** [svenerbe/ecoflow_dynamic_power_adjustment#36]
Hey everyone, 

I've had the script running wonderfully for about half a year now. Unfortunately, my Home Assistant crashed and I had no backup. 

Now I wanted to reinstall the script. But I just can't get it to feed me +/- 0W. When I activate the script, the inverter output goes up to 800W and then slowly goes back to 0W. After a few seconds do the same thing again. This goes all day long. 

Where is my mistake? 

I have attached my script again.

![Bildschirmfoto 2024-10-15 um 16 28 47 Kopie](https://github.com/user-attachments/assets/bdf3fbbc-029f-4324-a20e-2168890c4ec5)

**Comment by Burny08** [comment]
Edit: I found out something. Apparently my Bitshake read head doesn't react quickly enough and so the feed keeps adding up. 
For example: 
Current power consumption: 60W, house consumption increases every second: 60W, 120W, 180W, 240W, etc. 
until it notices that far too much is being fed in and immediately drops back down to 0W. 

I have now changed the interval of my bitshake from 10 seconds to 5 seconds and the script to 15 seconds. Now it seems to be working.

**Comment by Burny08** [comment]
Edit: I found something. 
Apparently my Bitshake read head doesn't react quickly enough and so the feed keeps adding up.

For example:
Current power consumption: 60W, house consumption increases every second: 60W, 120W, 180W, 240W, etc.
until it notices that far too much is being fed in and immediately drops back down to 0W.

I have now changed the interval of my bitshake from 10 seconds to 5 seconds and the script to 15 seconds. Now it seems to be working.
